src/test/java/com/dougnoel/sentinel/webdrivers/WebDriverFactoryTest.java

```diff
@@ -69,9 +69,15 @@ public void malFormedUrlTest() {
 		WebDriverFactory.instantiateWebDriver();
 	}
 	
-	@Test(expected = org.openqa.selenium.WebDriverException.class)
-	public void createCustomChromeDriverTest() {
-		System.setProperty("chromeBrowserBinary","fake/path");
+	@Test
+	public void createChromeOptionsChromeDriver() {
+		Configuration.update("chromeOptions", "start-maximized");
+		WebDriverFactory.instantiateWebDriver();
+	}
```

Comment:
The last line can be put in the after step.
```suggestion
	@Test
	public void createChromeOptionsChromeDriver() {
		Configuration.update("chromeOptions", "start-maximized");
		WebDriverFactory.instantiateWebDriver();
		var js = (JavascriptExecutor)Driver.getWebDriver();
		assertSame("Expecting window to be maximized.", "false", js.executeScript("return document.hidden").toString());
		WebDriverFactory.quit();
	}
```